schema/schema.cql

```diff
@@ -24,3 +24,9 @@ create table metric_name_set (
   metric_names set<varchar>,
   primary key (shard)
 );
+
+-- graphite_names
+create table graphite_names (
```

Comment:
Will sharding be useful here? The only operation that we can perform is getting *all* of them, since the Go code is handling the filtering. What would the benefit of splitting them up be?